Gemfile.lock

```diff
@@ -533,6 +526,7 @@ GEM
 
 PLATFORMS
   ruby
+  x86_64-linux
```

Comment:
What's this for?

Reply:
This fixed a problem the CI was having where bundler would crash and no tests would run. (it worked okay on my machine though)

Reply:
this makes me nervous as i don't really understand what it does but ok!